chia/daemon/server.py

```diff
@@ -699,6 +702,27 @@ async def register_service(self, websocket: WebSocketServerProtocol, request: Di
         log.info(f"{response}")
         return response
 
+    async def unregister_service(self, websocket: WebSocketServerProtocol, request: Dict[str, Any]) -> Dict[str, Any]:
+        self.log.info(f"Unregister service {request}")
+        service = request["service"]
+
+        connection_removed = False
+        if service in self.connections:
+            after_removal = []
+            for connection in self.connections[service]:
+                if connection == websocket:
+                    connection_removed = True
+                    continue
+                else:
+                    after_removal.append(connection)
+            self.connections[service] = after_removal
```

Comment:
Also looks the same as
```python
num_connections = len(self.connections.get(service, []))

if num_connections:
    self.connections[service] = [conn for conn in self.connections[service] if 
        conn != websocket
    ]

connection_removed = len(self.connections.get(service, [])) < num_connections
```

Reply:
Thanks for reviewing!
I've just submitted a new commit for reducing lines.